.github/CONTRIBUTING.md

```diff
@@ -15,29 +15,27 @@ Thanks for your interest in improving Web Chat. We invest heavily in engineering
 
 > If you need to build this project for customization purposes, we strongly advise you to refer to our [samples](https://github.com/microsoft/BotFramework-WebChat/tree/main/samples). If you cannot find any samples that fulfill your customization needs and you don't know how to do that, please [send your dream to us](https://github.com/microsoft/BotFramework-WebChat/issues/).
 >
-> Forking Web Chat to make your own customizations means you will lose access to our latest updates. Maintaining forks also introduces chores that are substantially more complicated than a version bump.
+> Forking Web Chat to make your own customizations means you will lose access to our latest features and security updates. Maintaining forks also introduces chores that are substantially more complicated than a version bump.
 
 # Preparing the environment
 
-There are 2 steps to prepare your environment: [install tools](#installing-tools) and [prepare the repository](#preparing-the-repository).
+There are 2 steps to prepare the environment: [install tools](#installing-tools) and [prepare the repository](#preparing-the-repository).
 
 ## Installing tools
 
-Please install the follow in your development environment:
+Please install the follow in the development environment:
 
 -  [Node.js](https://nodejs.org/) of LTS or latest version
--  [NPM] minimum required version v7.0.0-rc.3
+-  [NPM] with minimum version of 7.0.0
 -  [Docker](https://docs.docker.com/get-docker/)
-
-These are development environments we tested:
-
--  Windows 10 Professional
-   -  With WSL1 or [WSL2](https://aka.ms/wsl2) installed
--  [Ubuntu 18 running under WSL2](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q)
+-  On Windows 10, install [Windows Subsystem for Linux 2](https://aka.ms/wsl2)
+   -  [Ubuntu 20.04](https://www.microsoft.com/en-us/p/ubuntu-2004-lts/9n6svws3rx71)
 
 ## Preparing the repository
 
-Web Chat is a monorepo and uses [`lerna`](https://lerna.js.org/). After the repository is [forked and cloned](https://github.com/microsoft/botframework-webchat/fork), run the following to install all dependencies.
+To keep our main repository clean and easy to maintain, all changes must done in a [fork](https://github.com/microsoft/botframework-webchat/fork).
```

Comment:
Need to figure out how we're going to enforce this

Reply:
Seems no way to enforce this (for administrators/us).